### DISCUSSION TITLE
Can't upload files to tasmota file system

### DISCUSSION THREAD

**Discussion by Sum-Guy** [arendst/Tasmota#17575] [Q&A] [answered]
I've uploaded 1 file (display.ini) and I see there are 2 other (hidden) files on the file system, but I've been trying to upload another file (a bitmapped graphic file, about 60kb) and it doesn't seem to take it.  I've tried uploading random files (txt, etc) that are much smaller, and again they are not showing up after initiate the upload.  I'm not seeing anything on the serial log (no error messages).

Any ideas?

**Comment by nikito7** [comment]
No free ram.

Esp8266 or esp32?

**Comment by barbudor** [comment]
mini D1 with binary compiled from tasmoCompiler
No problem to load files of different size

![image](https://user-images.githubusercontent.com/2996042/210433692-ff5dbcf5-b198-4bda-8027-be18ce289859.png)

**Comment by Sum-Guy** [comment, marked as answer]
This was not an "out of memory" situation.  The file system had just under 2 mb of space, and the file being uploaded (that wasn't turning up) was about 60 kb.  Even smaller files (a few dozen bytes) weren't showing up.

This was on a lolin nodemcu, ESP8266EX, 4mb flash.

--------------------

I have erased and re-flashed the BIN file using esptool (esptool.exe v4.4) and the file system seems to work ok (I can upload multiple files without issues).

So I would say this issue would have been caused by flashing with tasmotizer.

So it seems this issue is closed.

**Comment by nikito7** [comment]
I can only upload files via webui if I disable my script and restart first. Esp8266.

How to use Curl?
